massa-bootstrap/src/server.rs

```diff
@@ -486,6 +486,7 @@ async fn manage_bootstrap(
                         Ok(Ok(_)) => Ok(()),
                     }?;
                 }
+                BootstrapClientMessage::BootstrapSuccess => return Ok(()),
```

Comment:
I think the server should print that the bootstrap succeeded

Reply:
So the BootstrapSuccess message is basically ignored server-side ?

Reply:
No it return from this function and go back to the upper scope which print.

Reply:
okay